Svc/FatalHandler/FatalHandlerComponentLinuxImpl.cpp

```diff
@@ -27,9 +27,9 @@ namespace Svc {
             FwEventIdType Id) {
         // for **nix, delay then exit with error code
         Fw::Logger::logMsg("FATAL %d handled.\n",(U32)Id,0,0,0,0,0);
+        (void)raise( SIGTERM );
         (void)Os::Task::delay(1000);
         Fw::Logger::logMsg("Exiting.\n",0,0,0,0,0,0);
```

Comment:
Would it help to add a message here?  e.g. "Exiting with segfault and core file"?

Reply:
Yes, I think so. Even if someone finds it suspicious, it will be easy enough to find such specific message in code to confirm that it's intentional.

Reply:
Would you like to make that change to line 32 and submit it here? Otherwise I will open up an issue to correct the message.